OrbitGl/ThreadTrack.cpp

```diff
@@ -139,17 +143,26 @@ void ThreadTrack::Draw(GlCanvas* canvas, PickingMode picking_mode) {
   event_track_->SetPos(pos_[0], pos_[1]);
   event_track_->SetSize(canvas->GetWorldWidth(), event_track_height);
   event_track_->Draw(canvas, picking_mode);
+
+  tracepoint_track_->SetPos(pos_[0], pos_[1]);
+  tracepoint_track_->SetSize(canvas->GetWorldWidth(), event_track_height);
+  tracepoint_track_->Draw(canvas, picking_mode);
 }
 
 void ThreadTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick, PickingMode picking_mode) {
   event_track_->SetPos(pos_[0], pos_[1]);
   event_track_->UpdatePrimitives(min_tick, max_tick, picking_mode);
+
+  tracepoint_track_->SetPos(pos_[0], pos_[1]);
+  tracepoint_track_->UpdatePrimitives(min_tick, max_tick, picking_mode);
+
   TimerTrack::UpdatePrimitives(min_tick, max_tick, picking_mode);
 }
 
 void ThreadTrack::SetEventTrackColor(Color color) {
```

Comment:
Consider renaming `SetEventTrackColor` to `SetTrackColor`.

Reply:
Of course! Since in that function the tracepoint track is also colored, it makes sense to change the name. Thank you!